backend/docker_ops.py

```
@@ -1,383 +1,643 @@
+            logger.info("Docker API connection established")
+        except DockerException as e:
+            logger.error(f"Failed to connect to Docker API: {e}")
+            raise
+    
+    def __del__(self):
+        """Clean up Docker client connection"""
+        if hasattr(self, 'client'):
+            try:
+                self.client.close()
+            except:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)